src/Android/Avalonia.AndroidTestApplication/Resources/Resource.Designer.cs

```diff
@@ -14,7 +14,7 @@ namespace Avalonia.AndroidTestApplication
 {
 	
 	
-	[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "1.0.0.0")]
+	[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "12.1.99.62")]
```

Comment:
nit: unrelated

Reply:
Grr visual studio keeps trying to sneak these in. Looks like this time it succeeded. Will revert.

Reply:
It's fine, otherwise VS will never shut up